heed/src/env.rs

```diff
@@ -278,22 +278,24 @@ pub enum CompactionOption {
 impl Env {
     /// Return the size used by all the databases in the environment.
     pub fn size(&self) -> Result<usize> {
```

Comment:
You should better use an u64 here as the disk space used can be bigger than the addressable space, right?

Reply:
I trust you on this one